skimage/measure/fit.py

```diff
@@ -492,66 +488,76 @@ def estimate(self, data):
         success : bool
             True, if model estimation succeeds.
 
-        """
 
+        References:
+        -----------
+            [1] Halir, R., Flusser, J.: 'Numerically Stable Direct Least Squares
+                Fitting of Ellipses'
```

Comment:
sorry, I cannot find DOI... [Numerically Stable Direct Least Squares Fitting Of Ellipses](https://www.researchgate.net/publication/2820897_Numerically_Stable_Direct_Least_Squares_Fitting_Of_Ellipses)